dictionary.md

```diff
@@ -247,10 +295,16 @@ Catching a query from the user and redirecting it to the appropriate API functio
 
 ### Runner
 
-An executable written in one of the supported programming languages that starts the [sequence](#sequence) created by developer, inside the container controlled by the [supervisor](#supervisor)
+An executable written in one of the supported programming languages that starts the [Sequence](#sequence) created by developer, inside the container
 
 ## S
 
+### Scalability
+<!-- TODO -->
+
+### Scope
+<!-- TODO -->
```

Comment:
Scope as in the CLI sessions?

Reply:
I guess so. In this case I would use a slightly edited version from https://github.com/scramjetorg/scramjet-cpm-dev/blob/main/cpm/packages/manager-cli/cli-draft.md#scope-help-output

```suggestion
A scope is a handy organization unit that lets a user switch between the currently used spaces and hubs. It is a named pair consisting of one space and one hub. This named pair is saved in the session config, which lets user work with different spaces and hubs sessions simultaneously. There can be multiple scopes, but only one can be used at once in a given session.
```